soroban-sdk/src/lib.rs

```diff
@@ -259,6 +262,52 @@ pub use soroban_sdk_macros::contracterror;
 /// ```
 pub use soroban_sdk_macros::contractimport;
 
+/// Marks a type as being the type that contract functions are attached for.
+///
+/// Use `#[contractimpl]` on impl blocks of this type to make those functions
+/// contract functions.
+///
+/// Note that a crate only ever exports a single contract. While there can be
+/// multiple types in a crate with `#[contract]`, when built as a wasm file and
+/// deployed the combination of all contract functions and all contracts within
+/// a crate will be seen as a single contract.
```

Comment:
This is interesting, and a little counter-intuitive. Do we currently have an example of a single crate with multiple `#[contract]`s built into a single contract?